src/App.js

```diff
@@ -4,12 +4,23 @@ function App() {
   return (
     <div className="App">
       <header className="App-header">My cakes</header>
-      <main className="App-main"></main>
+      <main className="App-main">
+        <ul>
+          <li className="Item-card">
+            <img src="" alt="" />
+            <h3 className="Item-card__name">Chocolate cake</h3>
```

Comment:
i know its like the first version of your structure, but here you could even use an custom component, and u just change the props of the word, since you are always repeating yourselve.

Reply:
Agree, was thinking the same but left it for the next branch !